### ISSUE TITLE
Can't resolve 'react-usefavicon' – are the main, browser etc properties in `package.json` correct?

### ISSUE THREAD

**Issue by philwolstenholme** [JulesBlm/useFavicon#5]
I was having trouble using `react-usefavicon` when trying:

```
import { useFavicon } from "react-usefavicon";
```

In the end I had to use this approach, but that means no automatically imported types:

```
import {
  drawTextBubble,
  useFavicon
} from 'node_modules/react-usefavicon/dist/react-usefavicon.modern.mjs';
```

Are the `browser`, `main` etc properties in `package.json` correct? It looks like they're missing a `react-` prefix in each of their filenames:

<img width="1276" alt="SCR-20230428-rqcs" src="https://user-images.githubusercontent.com/6339853/235235139-5a59d069-d8ef-4b93-bfc8-95055b92de9b.png">

**Comment by JulesBlm** [comment]
Good catch, thanks for making an issue. I just updated the export properties and published a new version. Could you check if that fixes it?

**Comment by JulesBlm** [comment]
Fixed in 2.0.0
